plugins/diagnostics/ts/diagnostics-jfr.controller.ts

```diff
@@ -238,14 +238,32 @@ namespace Diagnostics {
         for (let index = 0; index < existingRecordings.length; index++) {
           lastRecording = existingRecordings[index];
           if (lastRecording.state === "STOPPED") {
-            scope.recordings.push({
-              number: "" + lastRecording.id,
-              size: lastRecording.size + " b",
-              file: null,
-              time: lastRecording.stopTime,
-              canDownload: true,
-              downloadLink: jolokiaUrl + "/exec/jdk.management.jfr:type=FlightRecorder/downloadRecording(long)/" + lastRecording.id
-            });
+            if (jolokiaUrl.match("/actuator/hawtio/jolokia") != null) {
```

Comment:
For Hawtio on Spring Boot, the path `/actuator/hawtio/jolokia` is a redirect to `/actuator/jolokia`. So it's better to match `jolokiaUri` with `/actuator/jolokia` as well here.

Reply:
@tadayosi : I think the original /actuator/hawtio/jolokia is the fact that allows us to (hackishly) deduct that it is running in springboot hawtio. /actuator/jolokia could for instance also be part of the jolokia connection string _to_ a spring boot application, but hawtio running in a different stack ?

Reply:
Ah, yes I am thinking about the case where an user uses the Connect tab to connect from a different Hawtio to a remote Spring Boot app. When Hawtio is embedded in a SB app then `/actuator/hawtio/jolokia` should be just fine.